sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_generated/aio/operations/_operations.py

```diff
@@ -233,104 +125,9 @@ async def get_by_id(self, id: str, **kwargs: Any) -> AsyncIterator[bytes]:
             deserialized = response.iter_bytes()
 
         if cls:
-            return cls(pipeline_response, cast(AsyncIterator[bytes], deserialized), response_headers)
```

Comment:
Does the cls that we pass in here need to return an AsyncIterator[bytes] type? Or can it return something else?

Reply:
By design you don't control what cls returns

Reply:
so the return type will need to be ignored I think, will add a note too